Dockerfile

```diff
@@ -1,16 +1,14 @@
-FROM python:3.9-slim-bullseye
-
-RUN apt-get update \
-    && apt-get -y upgrade \
-    && apt-get install -y --no-install-recommends gcc python3-dev git \
-    && apt-get clean \
-    && rm -rf /var/lib/apt/lists/* \
-    && mkdir -p /usr/src/app
+FROM flask-base
 
+RUN mkdir -p /usr/src/app
 WORKDIR /usr/src/app
 
+<<<<<<< HEAD
+COPY ./container-sdx-controller /usr/src/app
+=======
 WORKDIR /usr/src/app
 COPY . /usr/src/app
+>>>>>>> main
```

Comment:
These changes to `Dockerfile` are not necessary, and they are not within the scope of this PR.  They contain errors too, with git merge markers. Please revert the changes.

Reply:
There is a different Dockerfile that needs to be used while running in compose mode. To avoid affecting the original code, I moved the file to a different directory. There was an error on Github, and although I did a merge, the file was not updated. Therefore, I will update it manually.

Reply:
This happens when there's conflict between your branch and your merged branch. You need to resolve the conflict between the branches.